oci/oci.go

```diff
@@ -116,6 +129,23 @@ func (r *Runtime) Name() string {
 // Depending if the container is privileged and/or trusted,
 // this will return either the trusted or untrusted runtime path.
 func (r *Runtime) Path(c *Container) string {
+	if c.runtimeHandler != "" {
+		runtimeHandler, ok := r.runtimes[c.runtimeHandler]
+		if !ok {
+			if c.runtimeHandler == UntrustedRuntime && r.untrustedPath != "" {
+				return r.untrustedPath
+			}
+
+			return r.trustedPath
+		}
+
+		if runtimeHandler.RuntimePath == "" {
+			return r.trustedPath
```

Comment:
how come this defaults to the trusted path?

Reply:
I guess that's fine. As long as the empty runtimeHandler case still defaults to the old behavior, which I think it does.